.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -34,6 +34,16 @@ body:
     description: 
     placeholder: |
       Any thoughts regarding how this should be implemented?
+- type: dropdown
```

Comment:
Perhaps instead of a user controlled field (which feels quite weak), we can do this:

1. Add a default label for all features that says "need discussion"
1. Add a note in the community notes section that says not to pick an item if it has that label.

Then, it is on us to make clear out this label when we groom the backlog (or schedule for discussion, request an RFC, etc).

Wdyt?

Reply:
I think most issues will not need this label, so it's better to do it the other way around: new issues are created without the label, and we add it if needed during triage.
That said, I'm also fine starting as you suggested.

Reply:
I am not sure I agree. Feels like it would be healthy for all *feature requests* to have some kind of review by someone from the team before they are ready to be implemented, no?

Reply:
I review everything anyway, so it's just a matter of adding the needs-discussion label or removing it.
Let's start with adding it automatically to alm issue. Fine by me